daily_milestone.go

```diff
@@ -225,12 +244,11 @@ func main() {
 		logger.Println(err)
 	}
 
-	newMilestones := createMilestoneData(advance)
-
+	newMilestones := createMilestoneData(advance, strings.ToLower(milestoneTime))
```

Comment:
Lowercasing is a nice solution for edge cases, but I assume you added it cause you are using "Daily" as default above. Probably best to just use "daily". It's what you are using in the explanation.